mlflow/genai/judges/utils.py

```diff
@@ -28,19 +28,77 @@
 
 # "endpoints" is a special case for Databricks model serving endpoints.
 _NATIVE_PROVIDERS = ["openai", "anthropic", "bedrock", "mistral", "endpoints"]
+_LITELLM_PROVIDERS = ["azure", "vertexai", "cohere", "replicate", "groq", "together"]
 
 # Global cache to track model capabilities across function calls
 # Key: model URI (e.g., "openai/gpt-4"), Value: boolean indicating response_format support
 _MODEL_RESPONSE_FORMAT_CAPABILITIES: dict[str, bool] = {}
 
 
+def _check_databricks_agents_installed(error_code: str = INVALID_PARAMETER_VALUE) -> None:
```

Comment:
I think we should always just use `BAD_REQUEST`; no need to parametrize by error code.